src/Api/MemberApi.php

```diff
@@ -77,7 +77,79 @@ public function subscribe(string $listId, Member $member): Member
                 '/%s/lists/%s/members/%s',
                 MailChimpConfig::VERSION,
                 $listId,
-                md5($member->getEmailAddress())
+                md5(strtolower($member->getEmailAddress()))
+            ),
+            [
+                'body' => $this->serializer->serialize($member, 'json')
+            ]
+        );
+
+        /** @var $successResponse Member */
+        $successResponse = $this->serializer->deserialize((string) $response->getBody(), Member::class, 'json');
+
+        return $successResponse;
+    }
+
+    /**
+     * Re Subscribe a member
+     *
+     * @param string $listId
+     * @param Member $member
+     * @return Member
+     * @throws ValidationFailedException
+     * @throws ClientException
+     */
+    public function reSubscribe(string $listId, Member $member): Member
```

Comment:
this method looks pretty similar with `subscribe()`. Any info what is the different on intent?

Reply:
This is `patch` method, on the other hand `subscribe()` is `put` method.

Reply:
We will need to think how to reduce code duplicate here. Currently `subscribe()`, `reSubscribe()` and `getStatus()` code content is similar and we can reduce it. Let's think about this in next iteration.